Mathlib/Order/CompleteLattice.lean

```diff
@@ -255,13 +255,23 @@ theorem infᵢ_le_iff {s : ι → α} : infᵢ s ≤ a ↔ ∀ b, (∀ i, b ≤
   simp [infᵢ, infₛ_le_iff, lowerBounds]
 #align infᵢ_le_iff infᵢ_le_iff
 
+-- Help wanted:
+-- This is showing on error on the `hy'` in `solve_by_elim [le_trans _ hy']`
+-- saying `unknown identifier 'hy''`.
+-- However `solve_by_elim` is working perfectly, successfully closing the goal using `hy'`.
+-- (See the #print statement below.)
+-- Who is generating this error, and why?
```

Comment:
@digama0, could I have some assistance diagnosing this error? I do not understand who is generating this error, when `solve_by_elim` seems to be successfully using this term!